### PR TITLE
feat: code style

### PR DESCRIPTION
**Pull request by aochmann**
**What this PR does / why it's submitted / why we need it**:

Added code style.

---

**Checklist**:

- [x] My PR has a descriptive title (not a vague title like `Needed this`)
- [x] Title and all the commits follow [commitlint guidelines](https://github.com/conventional-changelog/commitlint#what-is-commitlint)
- [ ] My PR targets the `develop` branch or appropriate `release` branch
- [ ] My PR provides test coverage for my change (where applicable)
- [x] I've placed the link to the corresponding Trello card or issue which this PR fixes/address below ⤵️

---

Fixes: https://github.com/thecogworks/Cogworks.AzureSearch/issues/29

### CODE REVIEW

File: src/Cogworks.AzureSearch/Operations/IndexOperation.cs [hunk 1]
```diff
@@ -29,7 +29,8 @@ public async Task<bool> IndexExistsAsync()
             }
             catch (Exception exception)
             {
-                throw new IndexExistsException(exception.Message, exception.InnerException);
+                var domainException = new IndexExistsException(exception.Message, exception.InnerException);
```
Comment (lkoruba) [inline src/Cogworks.AzureSearch/Operations/IndexOperation.cs:32]: What's the purpose of this split up?

Comment (aochmann) [reply src/Cogworks.AzureSearch/Operations/IndexOperation.cs:32]: this was also my question, I will link you to the article: https://cezarypiatek.github.io/post/exceptions-usages-analyzer/

in general I wanted to disable it in editorconfig but it looks like this analyzer does not support disabling rules. I will reply if I find something related with this split.

Comment (aochmann) [reply src/Cogworks.AzureSearch/Operations/IndexOperation.cs:32]: OK thats the case: https://cezarypiatek.github.io/post/exceptions-usages-analyzer/#context-will-save-you

and this rule make it failure: 
image

Comment (aochmann) [reply src/Cogworks.AzureSearch/Operations/IndexOperation.cs:32]: we cannot do to much - not including package downgrade -> but we don't want to do it.

**Comment by aochmann** [comment]
Close due to issues with branches. Will re-open new one.

**Pull request closed without merging**